src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -563,7 +563,7 @@ public static void _stopJenkins(Server server, List<LenientRunnable> tearDowns,
                 }
             }
         }
-
+        tweakXmlFileLogger();
```

Comment:
Loggers are generally static (as it the case in XmlFile) and as we are using the same JVM for a class rather than each test - does this not pollute the logger for the next test run in the same class, potentially causing a lot of log spam?

Reply:
You're right, should be better now